File: dart/lib/model/v3_outlet_geolocation_parameters.dart
part of swagger.api;

class V3OutletGeolocationParameters {
  /* Filter by maximum distance (in metres) from location specified via latitude and longitude parameters (default = 300) */
  double maxDistance = null;
/* Maximum number of results returned (default = 30) */
  int maxResults = null;

  V3OutletGeolocationParameters();

  @override
  String toString() {
    return 'V3OutletGeolocationParameters[maxDistance=$maxDistance, maxResults=$maxResults, ]';
  }

  V3OutletGeolocationParameters.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    maxDistance = json['max_distance'];
    maxResults = json['max_results'];
  }

  Map<String, dynamic> toJson() {
    return {
      'max_distance': maxDistance,
      'max_results': maxResults
     };
  }

  static List<V3OutletGeolocationParameters> listFromJson(List<dynamic> json) {
    return json == null ? new List<V3OutletGeolocationParameters>() : json.map((value) => new V3OutletGeolocationParameters.fromJson(value)).toList();
  }

  static Map<String, V3OutletGeolocationParameters> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, V3OutletGeolocationParameters>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new V3OutletGeolocationParameters.fromJson(value));
    }
    return map;
  }
}

File: csharp-dotnet2/src/main/CsharpDotNet2/IO/Swagger/Model/V3OutletGeolocationParameters.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class V3OutletGeolocationParameters {
    /// <summary>
    /// Filter by maximum distance (in metres) from location specified via latitude and longitude parameters (default = 300)
    /// </summary>
    /// <value>Filter by maximum distance (in metres) from location specified via latitude and longitude parameters (default = 300)</value>
    [DataMember(Name="max_distance", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "max_distance")]
    public double? MaxDistance { get; set; }

    /// <summary>
    /// Maximum number of results returned (default = 30)
    /// </summary>
    /// <value>Maximum number of results returned (default = 30)</value>
    [DataMember(Name="max_results", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "max_results")]
    public int? MaxResults { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class V3OutletGeolocationParameters {\n");
      sb.Append("  MaxDistance: ").Append(MaxDistance).Append("\n");
      sb.Append("  MaxResults: ").Append(MaxResults).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
